support/ci/appveyor.ps1

```diff
@@ -179,7 +179,7 @@ if((Test-DocsOnlyChanged -eq $true) -and (Test-ReleaseBuild -eq $false)) {
                     Copy-Item "/hab/pkgs/core/hab/*/*/bin/*" (Split-Path $habExe -Parent) -Force
                 }
             }
-            if(!(Test-PullRequest) -and $env:hab_components -ne "launcher") {
+            if(!(Test-PullRequest) -and $env:hab_components -eq "studio") {
```

Comment:
`$env:hab_components` will never equal `studio` rather it would be equal to `hab;plan-build-ps1;studio;sup;bintray-publish`. However what might be cleaner is to move the entire `if` block up just 2 lines to be inside of the component loop. then change to `if(!(Test-PullRequest) -and $component -eq "bintray-publish") {` this would make sure the container only gets built in a merge commit just after building `bintray-publish` which is really what you want.